.travis.settings.xml

```diff
@@ -0,0 +1,10 @@
+<settings>
```

Comment:
Go ahead and put this in buildscripts/ or similar. I'd rather not pollute the root folder.